include-what-you-use.1

```diff
@@ -243,6 +243,11 @@ Used after
 .B #include
 directives or forward declarations it ensures that they won't be removed.
 .TP
+.BR "// IWYU pragma: begin_keep"  , " // IWYU pragma: end_keep"
```

Comment:
Two spaces between `"` and comma, I think one is sufficient (unless there's something in man/troff syntax that requires it)